api/src/main/java/jakarta/mail/internet/MimeMessage.java

```diff
@@ -902,10 +909,8 @@ public Date getSentDate() throws MessagingException {
         String s = getHeader("Date", null);
         if (s != null) {
             try {
-                synchronized (mailDateFormat) {
-                    return mailDateFormat.parse(s);
-                }
-            } catch (ParseException pex) {
+                return Date.from(Instant.from(mailDateFormat.parse(s)));
```

Comment:
MailDateFormat has a concept of isLenient and per the API documents it was not clear to me if the default is lenient or strict.  I wonder if you need to use withResolverStyle to set that behavior and how compatible with existing behavior, mainly with parsing of dates.

Looking at the [MailDateFormatTest](https://github.com/jakartaee/mail-api/blob/master/api/src/test/java/jakarta/mail/internet/MailDateFormatTest.java) there are some tests that seem to imply leniency with lots of exceptions:

1. lenientMustBeBackwardCompatible()
2. mustUseLeniencyForParsing()
3. mustFailOrSkipCfws()
4. lenientMustAcceptNonMilitaryZoneNames()

I would think that for testing this version of the patch you would need to port the MailDateFormatTest into the MimeMessageTest.  Which seems like a lot of work.

What are your thoughts on just getting rid of the static field and the synchronized block and just creating some method local garbage by just creating a new MailDateFormat for parsing and formatting?